apps/opendosm/pages/dashboard/manufacturing-statistics/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { GetStaticProps, InferGetStaticPropsType } from "next";
+import { get } from "datagovmy-ui/api";
+import { Page } from "datagovmy-ui/types";
+import { Metadata } from "datagovmy-ui/components";
+import { useTranslation } from "datagovmy-ui/hooks";
+import { withi18n } from "datagovmy-ui/decorators";
+import { AnalyticsProvider } from "datagovmy-ui/contexts/analytics";
+import ManufacturingStatisticsDashboard from "@dashboards/manufacturing-statistics";
+
+const ManufacturingStatistics: Page = ({
+  last_updated,
+  meta,
+  timeseries,
+  timeseries_callout,
+}: InferGetStaticPropsType<typeof getStaticProps>) => {
+  const { t } = useTranslation(["dashboard-manufacturing-statistics", "common"]);
+  return (
+    <AnalyticsProvider meta={meta}>
+      <Metadata title={t("header")} description={t("description")} keywords={""} />
+      <ManufacturingStatisticsDashboard
+        last_updated={last_updated}
+        timeseries={timeseries}
+        timeseries_callout={timeseries_callout}
+        meta={meta}
+      />
+    </AnalyticsProvider>
+  );
+};
+
+export const getStaticProps: GetStaticProps = withi18n(
+  "dashboard-manufacturing-statistics",
+  async () => {
+    const { data } = await get("/dashboard", { dashboard: "manufacturing_statistics" });
+
+    return {
+      notFound: false,
+      props: {
+        meta: {
+          id: "dashboard-manufacturing-statistics",
+          type: "dashboard",
+          category: "services",
+          agency: "bppib",
```

Comment:
the meta prop here is actually catered for analytics. So, it should return `dosm`

While it may be helpful if the agency matches with the AgencyBadge prop, in this case, it isnt. Since AgencyBadge displays the division level unit inside DOSM. Same goes with future dashboards later in opendosm